static/app/views/routeError.tsx

```diff
@@ -80,6 +81,7 @@ function RouteError({error, disableLogSentry, disableReport, project, routes}: P
     return function cleanup() {
       window.clearTimeout(reportDialogTimeout);
     };
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Why is this type of ignores needed? I thought you were focusing on `no-restricted-imports`